src/main/scala/no/ndla/articleapi/model/api/TSTypes.scala

```diff
@@ -0,0 +1,15 @@
+package no.ndla.articleapi.model.api
+
+import com.scalatsi._
+import no.ndla.articleapi.model.domain
+
+/**
+  * The `scala-tsi` plugin is not always able to derive the types that are used in `Seq` or other generic types.
+  * Therefore we need to explicitly load the case classes here.
+  * This is only necessary if the `sbt generateTypescript` script fails.
```

Comment:
Denne gjelder ikke her heller :smile:

Reply:
Nja ish, måtte fortsatt ha `Either` typen definert:
```
implicit val relatedContent: TSIType[RelatedContentLink] = TSType.fromCaseClass[RelatedContentLink]
```
 for at sbt generate skulle kjøre, så tenkte at jeg kunne la kommentaren stå, siden den beskriver hvorfor ting er definert her.

Reply:
Nvm, så ditt kommentar forslag, bruker heller det :+1: